### PR TITLE
fix: add /provider directory for dependabot

### PR DESCRIPTION
**Pull request by dirien**
Hi,

This PR fixes the directory for the dependabot, when looking for go modules.

### CODE REVIEW

**Review by ringods — CHANGES_REQUESTED** [review]
@dirien can you copy the `gomod` section for the `sdk` folder too?

**Comment by dirien** [comment]
@ringods ofc! Done!

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
     labels:
       - "dependencies"
     commit-message:
-      prefix: "feat"
+      prefix: "chore"
+      include: "scope"
+  - package-ecosystem: "gomod"
+    directory: "/sdk"
+    schedule:
+      interval: "daily"
```
Comment (ringods) [inline .github/dependabot.yml:16]: If you don't want to be overloaded with PRs for every Pulumi release, I would lower the interval to monthly.

**Review by ringods — DISMISSED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (dirien) [reply .github/dependabot.yml]: Good point! I changed it to `weekly` on a `Sunday`. I am afraid to miss a patch of a critical vulnability.

**Review by ringods — APPROVED** [review]

**Pull request merged**